day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,65 @@
+# frog_number = {
+#   1 => "one",
+#   2 => "two",
+#   3 => "three",
+#   4 => "four",
+#   5 => "five",
+#   6 => "six",
+#   7 => "seven",
+#   8 => "eight",
+#   9 => "nine",
+#   10 => "10"
+# }
+
+frog_number = {
+  10 => "ten",
+  9 => "nine",
+  8 => "eight",
+  7 => "seven",
+  6 => "six",
+  5 => "five",
+  4 => "four",
+  3 => "three",
+  2 => "two",
+  1 => "one"
+}
+
+def plural_frog(key)
+  (key == 1) ? "frog" : "frogs"
+end
+
+def plural_frogs(number_2)
+  (number_2 == 1) ? "frog" : "frogs"
+end
+
+def were_was(number_2)
+  (number_2 == 1) ? "was" : "were"
+end
+
+
+# whole_number = (1..10).to_a.reverse
+
+
+frog_number = frog_number.map do |key, value|
```

Comment:
Good instinct to use iteration and nice work using the hash block variable syntax with `|key, value|`. 

You don't gain anything by reassigning `frog_number` on this line or using `.map` instead of `.each`. Take a look at the difference between those two enumerators to see why you'd get the same output with `.each` in this implementation.